resources/deployer/deployer.sh

```diff
@@ -9,10 +9,13 @@ DEPLOYMENTS_DIR=${DEPLOYER_DEPLOYMENTS_DIR:="$CRAFTER_DATA_DIR/repos/sites"}
 TARGETS_DIR=$DATA_DIR/targets
 PROCESSED_COMMITS=$DATA_DIR/processed-commits
 PORT=${DEPLOYER_PORT:="9191"}
-JAVA_OPTS="$DEPLOYER_JAVA_OPTS -Dtomcat.url=$TOMCAT_URL -Des.url=$ES_URL -Dmail.host=$MAIL_HOST \
-  -Dmail.port=$MAIL_PORT -Dserver.port=$PORT -Dlogging.config=$DEPLOYER_HOME/logging.xml -Dlogs.dir=$LOGS_DIR \
-  -Ddeployments.dir=$DEPLOYMENTS_DIR -Dtargets.dir=$TARGETS_DIR -DprocessedCommits.dir=$PROCESSED_COMMITS \
-  -Dloader.path=$DEPLOYER_HOME/lib"
+ENGINE_URL=${ENGINE_URL:="http://localhost:8080"}
```

Comment:
Why not extract all URLs to a single external file, say `set-urls.sh` and put everything in one clear place for anyone to update?